.github/workflows/pytest.yml

```diff
@@ -0,0 +1,25 @@
+name: Python test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
```

Comment:
We probably only need 3.5 (16.04) and 3.6 (18.04). If this helps to ease things.

Reply:
I'd rather keep 2.7 just for testing purposes. 3.7 is removed.